modules/redis/charts.go

```diff
@@ -0,0 +1,190 @@
+package redis
+
+import "github.com/netdata/go-orchestrator/module"
+
+type (
+	// Charts is an alias for module.Charts
+	Charts = module.Charts
+	// Dims is an alias for module.Dims
+	Dims = module.Dims
+)
+
+const (
+	keysRedisChartId = "keys_redis"
+)
+
+var charts = Charts{
+	{
+		ID:    "operations",
+		Title: "Operations",
+		Units: "operations/s",
+		Fam:   "operations",
+		Ctx:   "redis.operations",
+		Type:  module.Line,
+		Dims: Dims{
+			{
+				ID:   "total_commands_processed",
+				Name: "commands",
+				Algo: module.Incremental,
+			},
+			{ID: "instantaneous_ops_per_sec", Name: "operations"},
+		},
+	},
+	{
+		ID:    "hit_rate",
+		Title: "Hit rate",
+		Units: "percentage",
+		Fam:   "hits",
+		Ctx:   "redis.hit_rate",
+		Type:  module.Line,
+		Dims: Dims{
+			{ID: "hit_rate", Name: "rate"},
+		},
+	},
+	{
+		ID:    "memory",
+		Title: "Memory utilization",
+		Units: "KiB",
+		Fam:   "memory",
+		Ctx:   "redis.memory",
+		Type:  module.Line,
+		Dims: Dims{
+			{
+				ID:   "used_memory",
+				Name: "total",
+				Div:  1024,
+			},
+			{
+				ID:   "used_memory_lua",
+				Name: "lua",
+				Div:  1024,
+			},
+		},
+	},
+	{
+		ID:    "net",
+		Title: "Bandwidth",
+		Units: "kilobits/s",
+		Fam:   "network",
+		Ctx:   "redis.net",
+		Type:  module.Area,
+		Dims: Dims{
+			{ID: "total_net_input_bytes", Name: "in", Algo: module.Incremental, Mul: 8, Div: 1000},
+			{ID: "total_net_output_bytes", Name: "out", Algo: module.Incremental, Mul: -8, Div: 1000},
+		},
+	},
+	{
+		ID:    keysRedisChartId,
+		Title: "Keys per Database",
+		Units: "keys",
+		Fam:   "keys",
+		Ctx:   "redis.keys",
+		Type:  module.Line,
+	},
+	{
+		ID:    "keys_pika",
+		Title: "Keys",
+		Units: "keys",
+		Fam:   "keys",
+		Ctx:   "redis.keys",
+		Type:  module.Line,
+		Dims: Dims{
+			{ID: "kv_keys", Name: "kv"},
+			{ID: "hash_keys", Name: "hash"},
+			{ID: "list_keys", Name: "list"},
+			{ID: "zset_keys", Name: "zset"},
+			{ID: "set_keys", Name: "set"},
+		},
+	},
+	{
+		ID:    "eviction",
+		Title: "Evicted Keys",
+		Units: "keys",
+		Fam:   "keys",
+		Ctx:   "redis.eviction",
+		Type:  module.Line,
+		Dims: Dims{
+			{ID: "evicted_keys", Name: "evicted"},
+		},
+	},
+	{
+		ID:    "connections",
+		Title: "Connections",
+		Units: "connections/s",
+		Fam:   "connections",
+		Ctx:   "redis.connections",
+		Type:  module.Line,
+		Dims: Dims{
+			{ID: "total_connections_received", Name: "received", Algo: module.Incremental, Mul: 1},
```

Comment:
`Mul: 1` is default, no reason to explicitly use it

Reply:
Sure, no problem with sticking with the defaults here and in the other places (i.e. `Mul`, `Line`...) :+1:

Reply:
I'm ok with stating things explicitly. Explicit > Implicit :)

Reply:
That would be my personal preference as well, I think it makes things more readable but in this case I don't mind either way. Up to you guys.

Reply:
I agree that explicit > implicit in general, but not in this case.

>  I think it makes things more readable

In my opinion we will get more noise then readability, so it is not worth it.